blockchain/chain/src/store/chain_store.rs

```diff
@@ -415,7 +415,7 @@ where
 
     async fn parent_state_tsk<'a>(
         &'a self,
-        key: &TipsetKeys,
+        key: &'a TipsetKeys,
```

Comment:
Should the tipset keys have the `'a` lifetime?

Reply:
Probably there is a more elegant solution, but I couldn't find one

Reply:
Are you sure it fails to compile if you remove the lifetime variable? If so, please paste the error message here.

Reply:
It vanished, so I deleted the lifetime